src/Application/Injectors/FileProcessors.php

```diff
@@ -25,28 +20,9 @@ final class FileProcessors
     public function __invoke(): array
     {
         return [
-            FileProcessor::class => static function () {
-                $container = EcsContainer::make();
-
-                $reflection = new Reflection($container->get(SniffFileProcessor::class));
-
-                $fixer = $reflection->get('fixer');
-                $errorAndDiffCollector = $reflection->get('errorAndDiffCollector');
-                $appliedCheckersCollector = $reflection->get('appliedCheckersCollector');
-                /** @var \Symplify\EasyCodingStandard\Console\Style\EasyCodingStandardStyle $easyCodingStandardStyle */
-                $easyCodingStandardStyle = $container->get(EasyCodingStandardStyle::class);
-                /** @var \Symplify\EasyCodingStandard\Skipper $skipper */
-                $skipper = $container->get(Skipper::class);
-
-                return new FileProcessor(
-                    $fixer,
-                    new FileFactory(
-                        $fixer,
-                        $errorAndDiffCollector,
-                        $skipper,
-                        $appliedCheckersCollector,
-                        $easyCodingStandardStyle
-                    )
+            SniffFileProcessor::class => static function () {
```

Comment:
All of the ECS hacking removed and now the file processor just need a file factory